src/components/FilterButtons.jsx

```diff
@@ -0,0 +1,24 @@
+import Button from 'components/ui/Button';
+import { LABEL_ALL, LABEL_COMPLETE, LABEL_INCOMPLETE } from 'utils/constants';
+
+function FilterButtons() {
+  const filterButtons = [
```

Comment:
variable name and component name are same

Reply:
renamed as `actionButtons`